src/Carousel.tsx

```diff
@@ -542,6 +550,7 @@ function CarouselFunc(uncontrolledProps: CarouselProps, ref) {
       <div className={`${prefix}-inner`}>
         {map(children, (child, index) => {
           const isActive = index === renderedActiveIndex;
+          intervals.current[index] = child.props.interval as number;
```

Comment:
put this in the `map` clause where we count `numChildren`

Reply:
https://github.com/react-bootstrap/react-bootstrap/blob/3b0091002defe502c69a7c5a7ab09e75c090e13c/src/Carousel.tsx#L270-L272
Hmm, I'm not exactly sure where the `map` clause would be here... I'm probably missing something obvious 😆

Reply:
in https://github.com/react-bootstrap/react-bootstrap/blob/ba7d0f3a8e300b8b09d37f575fed51fac2b6f725/src/Carousel.tsx#L275-L277 – change that to a `forEach` or something instead of a `.length`, and extract both the current interval _and_ the number of children